app/src/main/res/layout/activity_album_review.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:orientation="vertical"
+              android:weightSum="1"
+              android:background="@drawable/album_border"
+              android:padding="@dimen/album_art_border">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/album_art_size"
+        android:orientation="vertical"
+        android:background="@drawable/album_border"
+        android:padding="@dimen/album_info_border">
+
+        <TextView
+            android:id="@+id/textview_album_name_review"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_weight="0.5"
+            android:background="@color/colorPrimary"
+            android:gravity="center"
+            android:textColor="@android:color/black"
+            android:textSize="@dimen/list_font_size"
+            android:textStyle="bold"
+            android:padding="@dimen/album_art_border"
+            tools:text="Album Name"/>
+
+        <TextView
+            android:id="@+id/textview_artist_name_review"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_gravity="left"
+            android:layout_weight="0.5"
+            android:background="@color/colorPrimary"
+            android:gravity="center"
+            android:textColor="@android:color/white"
+            android:textSize="@dimen/list_font_size"
+            android:textStyle="bold"
+            tools:text="Artist Name"/>
+    </LinearLayout>
+
+    <ImageView
+        android:id="@+id/imageview_album_art_review"
+        android:layout_height="@dimen/album_art_size"
+        android:layout_width="@dimen/album_art_size"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_centerVertical="true"
+        android:src="@mipmap/ic_launcher"
+        android:layout_gravity="center" />
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@drawable/album_border"
+        android:padding="@dimen/album_info_border">
+
+        <FrameLayout
```

Comment:
Getting fairly detailed here but you could probably avoid the nested views here by using a layer list in your album_border shape xml:

```
<layer-list xmlns:android="http://schemas.android.com/apk/res/android">

    <item>
        <shape android:shape="rectangle">
            <solid android:color="@color/colorPrimary" />
        </shape>
    </item>
    <item>
        <shape xmlns:android="http://schemas.android.com/apk/res/android"
            android:background="@color/colorPrimary"
            android:shape="rectangle">
            <corners
                android:bottomLeftRadius="0dp"
                android:bottomRightRadius="0dp"
                android:radius="2dp"
                android:topRightRadius="0dp" />
            <stroke
                android:width="3dp"
                android:color="@android:color/black" />
        </shape>
    </item>

</layer-list>
```